ajax/libs/slim.js/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "slim.js",
+  "filename": "slim.js",
```

Comment:
@hare1039 you should ask if author can provide minified file first, since the author is here.